dadi/lib/api/index.js

```diff
@@ -357,7 +366,16 @@ function notFound (api, req, res) {
       path[0].handler(req, res)
     } else {
       // otherwise, respond with default message
```

Comment:
@jimlambie do we need `res.setHeader('Content-Type', 'text/html')` here as L341?